src/js/components/RadioButton/RadioButton.js

```diff
@@ -100,7 +100,13 @@ const RadioButton = forwardRef(
             })}
           />
           {children ? (
-            children({ checked, focus: focus && focusIndicator, hover })
+            <StyledRadioButtonBox
+              focus={focus && focusIndicator && usingKeyboard}
```

Comment:
since we are adding another `box` here as well as making `focus` applied in every scenario this wouldnt be backwards compatible. Since the issue is just Star rating and thumb rating having the focus I wonder if we can adjust the code there in those files since we are passing icons as children we can pull out the `focus` and set it in the thumbRating and starRating files.